spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+RSpec.describe "Guest show page" do
+  let(:hotel) { Hotel.create(name: "Continental", location: "New York") }
+  let(:guest) { Guest.create(name: "John Wick", nights: 3) }
+  let(:room) { Room.create(hotel: hotel, suite: "President Suite", rate: 100) }
+  let!(:room_guest) { RoomGuest.create(guest: guest, room: room) }
+
+  it "should show guest name and list all of the rooms" do
+    visit "/guests/#{guest.id}"
+
+    expect(page).to have_content("John Wick")
+    expect(page).to have_content("Rooms stayed in:")
+
+    within("#rooms") do
+      expect(page).to have_content("Suite: President Suite")
+      expect(page).to have_content("Rate: 100")
+      expect(page).to have_content("Hotel: Continental")
+    end
+  end
```

Comment:
It would be good to get some more rooms for the guest so you can make sure ALL of their rooms can get displayed, and that it doesnt just display one room.